lang/kotlin/kotlin.py

```diff
@@ -0,0 +1,166 @@
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+ctx.matches = r"""
+code.language: kotlin
+"""
+
+
+@ctx.action_class("user")
+class UserActions:
+    def code_comment_line_prefix():
+        actions.insert("// ")
+
+    def code_operator_lambda():
+        actions.auto_insert(" -> ")
+
+    def code_operator_subscript():
+        actions.insert("[]")
+        actions.key("left")
```

Comment:
should this also be using `actions.user.insert_between` ?

Reply:
Oops I missed that one. Fixed now.